www/styles/admin-swat-local.css

```diff
@@ -176,28 +304,45 @@ span.swat-checkbox-wrapper {
 span.swat-checkbox-shim {
 	-webkit-appearance: none;
 	-moz-appearance: none;
+	z-index: 2;
 	display: inline-block;
-	width: 18px;
-	height: 18px;
+	width: 16px;
+	height: 16px;
 	vertical-align: middle;
 	margin: 3px 3px 3px 4px;
 	text-indent: -25px;
-	border: none;
+	border: 0;
 	padding: 0;
 	background: url(../images/admin-checkbox-radio-buttons.png) no-repeat;
```

Comment:
I know they aren't introduced by this PR, just re-styled, but why do we have custom radio/checkbox input images at all? Why not just rely on the browser defaults?